### PR TITLE
Zp integer

### PR DESCRIPTION
**Pull request by ZachParris**
Basic Integer Class with unit tests. : ) @katerebekah @BlaiseGratton @jcockhren @mattbruton

### CODE REVIEW

File: Mathmagician/Commands/Integer.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace Mathmagician.Commands
+{
+    public class Integer : Command
+    {
+        public override void CreateNumbers(int quantityOfNumsToGenerate)
+        {
+            //Create integers to add to Numbers List
+            for (int i = 0; i != quantityOfNumsToGenerate; i++)
+            {
+                Numbers.Add(i);
```
Comment (mattbruton) [inline Mathmagician/Commands/Integer.cs:16]: Looks good, could also implement a way to incorporate negative integers as well, but not a huge deal!

File: Mathmagician.Tests/IntegerTests.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        [TestMethod]
+        public void IntegerOutputQuantityMatchesUserInput()
+        {
+            Integer outputInts = new Integer();
+            outputInts.CreateNumbers(6);
+
+            Assert.AreEqual(6, outputInts.Numbers.Count);
+        }
+
```
Comment (mattbruton) [inline Mathmagician.Tests/IntegerTests.cs:36]: Maybe create a test to see if Integer can create integers? The other tests seem good to me.

**Comment by mattbruton** [comment]
The note about negative integers is purely optional, but I would like to see the test I mentioned implemented before I can give you the thumb @ZachParris

**Comment by ZachParris** [comment]
Right on! Thanks Matt! Wrote the test, and I'm planning on incorporating negative ints if time permits today. All super helpful tho, thank you!

File: Mathmagician/Commands/Even.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace Mathmagician.Commands
+{
+    class Even : Command
+    {
+        public override void CreateNumbers(int quantityOfNumsToCreate)
+        {
+            for (var i = 0; Numbers.Count != quantityOfNumsToCreate; i++)
+            {
+
```
Comment (mattbruton) [inline Mathmagician/Commands/Even.cs]: You're missing something here. Right now, this method just counts, but doesn't affect the list in Even.

**Comment by mattbruton** [comment]
👍 Good job, man. Feel free to merge whenever an instructor gets to look it over

File: Mathmagician/Commands/Integer.cs (hunk 1 above)
Comment (katerebekah) [inline Mathmagician/Commands/Integer.cs:16]: If this is fed a negative number, it will loop forever

**Comment by katerebekah** [comment]
👍

File: Mathmagician/Commands/Integer.cs (hunk 1 above)
Comment (mattbruton) [inline Mathmagician/Commands/Integer.cs:16]: I should have been more clear, I meant generating negative integers, but not using them as 'i'. The idea I had in my head is probably too much work for the exercise, but would involve using the for loop to create the initial number and then randomly make them a positive or negative number before adding to the list.

**Pull request merged**